### ISSUE TITLE
Update tC to handle ellipses (instead of `space-three periods-space`) in tN

### ISSUE THREAD

**Issue by benjore** [unfoldingWord/translationCore#6222]
### User Story

As a creator of tN, I want to use either an ellipses or a "space-ellipsis-space" so that I can more easily edit the tN, especially in Hebrew.

### Features / Specifications

- [ ] Update tN parser
- [ ] Update CSV exporter
- [ ] tN parser should handle
  - [x] ellipsis
  - [x] dot-dot-dot
  - [x] space-ellipsis-space
  - [x] space-dot-dot-dot-space
- [ ] CSV export will mirror whatever was in the original file
[en_tn_41-MAT.tsv.zip](https://github.com/unfoldingWord-dev/translationCore/files/3379015/en_tn_41-MAT.tsv.zip)

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by mannycolon** [comment]
## Spike Findings

Removing spaces before and after ellipsis could significantly break the TSV parser. Major changes would have to be made that I can't even quantify at this point. The space before and after the ellipsis actually helps in finding the word occurrence number.

However, a potential solution would be adding code to the `tsv-parser` to add spaces before and after the ellipsis right before starting the parsing process.

**Comment by elsylambert** [comment]
On Hold until the content team publishes the new tsv content.

**Comment by mannycolon** [comment]
This issue is good to go now. The new tN content was published about an hour ago.

**Comment by cckozie** [comment]
2.0.0 (bd37460) 
I'm seeing the ellipses character in the following checks in Titus:
- Metaphors: 1:3, 1:15, 
- Personification: 2:11 (actually has two ellipses together), 3:4
- Names: 3:12 (No alignments so check indicates content problem)
They are all rendering correctly in scripture pane

Also ran these tests:
- Added spaces before and after the ellipses character: Rendered correctly in scripture pane
- Replaced ellipses character with three periods: Rendered correctly in scripture pane
- Replaced ellipses character with space period period period space: Rendered correctly in scripture pane

On CSV export, the original language quote always has spaces around both the 3 dots and the ellipses. @benjore - Should I fail the issue for the extra spaces?

**Comment by benjore** [comment]
Since that info cannot be rounds tripped back into the app, it doesn't matter if there is a space.

**Comment by benjore** [comment]
Story #6200 will fix this problem.

**Comment by elsylambert** [comment]
Verified in 2.0.0 (8f84296).  Ellipsis character is working fine for tN. Checked in NT Book- PHM and OT book- RUT.
